de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/functions/exportSalesInvoicesInDateRange.sql

```diff
@@ -0,0 +1,38 @@
+DROP FUNCTION IF EXISTS exportSalesInvoicesInDateRange(TIMESTAMP With Time Zone, TIMESTAMP With Time Zone, numeric);
+
+CREATE OR REPLACE FUNCTION exportSalesInvoicesInDateRange(IN p_C_Invoice_dateinvoiced_start TIMESTAMP With Time Zone, 
+														  IN p_C_Invoice_dateinvoiced_end TIMESTAMP With Time Zone,
+														  IN p_AD_Org_ID numeric)
+
+    RETURNS TABLE
+            (
+                AD_Org_ID character varying,
+				C_Bpartner_ID character varying,
+				C_Invoices_Count bigint
+            )
+AS
+
+$BODY$
+
+SELECT DISTINCT org.name as AD_Org_ID,
+	bp.name as C_Bpartner_ID,
+	(SELECT COUNT(*)
```

Comment:
please rewrite the select to get rid of the subselect, and have just one where-clause.
you can do this with "group by"